values.yaml

```diff
@@ -60,15 +60,23 @@ global:
   # repository: the name of the docker image repository that is the source of local application images that was populated during the project build process and will now be referenced in the pods
   repository: palisade
 
+  # deployment: [codeRelease | example | <any previous code release tag>] the target deployment mode for partitioning external type dependencies, the value "codeRelease" will align it with the java parent project version
+  deployment: codeRelease
+
   # see charts/kafka/charts/zookeeper
   zookeeperConnectString: palisade-zookeeper:2181
 
   # persistent-volume and associated claims configurations
   persistence:
-    storageClassDeploy: false
-    storageClassName: efs-sc
+    storageClassDeploy: true
+#   storageClassName: efs-sc
```

Comment:
Is the storageClassName still used? If not can we delete this line

Reply:
no - I'll remove it